Detectors/TPC/calibration/src/CalibPedestalParam.cxx

```diff
@@ -0,0 +1,18 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HwClustererParam.cxx
+/// \brief Implementation of the parameter class for the hardware clusterer
+/// \author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+
+#include "TPCCalibration/CalibPedestalParam.h"
+
+using namespace o2::tpc;
```

Comment:
is this obsolete?

Reply:
uups. No, only not yet implemented since I stumbled about the decoder problems which were fixed meanwhile. I'll do this now.

Reply:
done